src/index.test.ts

```diff
@@ -3714,40 +3714,78 @@ describe('normalize', function () {
   });
 });
 
-it('personalSign and recover', function () {
-  const address = '0x29c76e6ad8f28bb1004902578fb108c507be341b';
-  const privKeyHex =
-    '4af1bceebf7f3634ec3cff8a2c38e51178d5d4ce585c52d6043e5e2cc3418bb0';
-  const privKey = Buffer.from(privKeyHex, 'hex');
-  const message = 'Hello, world!';
-  const msgParams: sigUtil.MsgParams<string> = { data: message };
-
-  const signed = sigUtil.personalSign(privKey, msgParams);
-  msgParams.sig = signed;
-  const recovered = sigUtil.recoverPersonalSignature(
-    msgParams as sigUtil.SignedMsgParams<string>,
+describe('personalSign', function () {
+  const privateKey = Buffer.from(
+    '4af1bceebf7f3634ec3cff8a2c38e51178d5d4ce585c52d6043e5e2cc3418bb0',
+    'hex',
   );
+  const helloWorldSignature =
```

Comment:
Isn't this a good place for a snapshot? I'm genuinely curious – I've never used snapshots myself.

Reply:
It would be, except that this is the input for the recovery tests.

Generally snapshots are great for cases where we want to ensure the output doesn't change, and where the output isn't readable. Even then, inline snapshots should be preferred for shorter outputs, so that it's easier to see what's going on when they change. I try to use them as a last resort, but I did find them very useful for opaque signatures.

Reply:
Ah, I didn't pay close enough attention to where it was used elsewhere. Thanks!